.ci/deploy.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -eo pipefail # Exit with nonzero exit code if anything fails
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+# Pull requests and commits to other branches shouldn't try to deploy, just build to verify
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "$SOURCE_BRANCH" ]; then
+    echo "Skipping deploy."
+    exit 0
+fi
+
+if [ -z "$ENCRYPTION_LABEL" ]; then
```

Comment:
this makes `[ "$TRAVIS_PULL_REQUEST" != "false" .. ` irrelevant...?

PRs to any repo with `ENCRYPTION_LABEL` set:

- dont get that `ENCRYPTION_LABEL` set, or
- any manual addition of it in `.travis.yml` is malicious/stupid, but the decrypt part of the deploy below will fail.

Reply:
`ENCRYPTION_LABEL` is not something that must be kept in hidden env var, although I put it there.